app/lang/en/users.php

```diff
@@ -52,6 +52,6 @@
   'updated_error'  => 'There was a problem updating that account.',
   'registration' => array(
       'thanks'       => 'Thanks for signing up to use Learning Locker. To complete your registration, we need you to verify your email.',
-      'instructions' => 'Please go to the following web address:'
```

Comment:
Ah sorry @davidpesce, I thought you'd intended to change the code that uses this. I'm actually not sure whether it's better to change it here or there. If you're certain that "instructions" hasn't been used anywhere else then this is fine.